tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -487,6 +487,46 @@ TEST (nnstreamerFilterTvm, launch01_n)
   g_free (model_file);
 }
 
+/**
+ * @brief Negative case with invalid custom properties
+ */
+TEST (nnstreamerFilterTvm, launch02_n)
```

Comment:
let's separate this into two cases :)
launchInvalidNum() and launchInvalidDev()

Reply:
I split it, PTAL.